File: tourneyAPI/Models/Entities/Characters/Sephiroth.cs
namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class Sephiroth : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Sephiroth()
    {
        Id = CharacterId.Sephiroth;
        characterName = CharacterName.SEPHIROTH;
        archetype = Archetype.BAIT_AND_PUNISH;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.FEATHERWEIGHT;
        tierPlacement = TierPlacement.B;
    }

}

File: tourneyClient/src/models/entities/Characters/Sephiroth.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Sephiroth: Character =
{
    id: CharacterId.Sephiroth,
    characterName: CharacterName.SEPHIROTH,
    archetype: Archetype.BAIT_AND_PUNISH,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.FEATHERWEIGHT,
    tierPlacement: TierPlacement.B
}

export default Sephiroth; 